support/build.gradle

```diff
@@ -24,3 +36,44 @@ test {
         exceptionFormat = 'full'
     }
 }
+
+version = VERSION_NAME
+
+task sourcesJar(type: Jar) {
+    from sourceSets.main.allSource
+    classifier = 'sources'
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    classifier = 'javadoc'
+    from javadoc.destinationDir
+}
+
+artifacts {
+    archives javadocJar
+    archives sourcesJar
+}
+
+bintray {
+    /*
+    These values can be found on https://bintray.com/profile/edit
+    BINTRAY_USER : your personal profile name (from "Your Profile")
+    BINTRAY_KEY : found on the left menu, under "API Key"
+     */
+    user = System.getenv('BINTRAY_USER')
+    key = System.getenv('BINTRAY_KEY')
+
+    configurations = ['archives']
+    pkg {
+        userOrg = 'shopify'
+        repo = 'shopify-java'
```

Comment:
The only repo we have is `shopify-android` which didn't seem like an appropriate name for a project that isn't shopify specific.  Although, I'm also not sure how to create a new repo under the Shopify organization on bintray.